core/src/main/java/io/confluent/kafka/schemaregistry/metrics/SchemaRegistryMetric.java

```diff
@@ -16,31 +16,50 @@
 package io.confluent.kafka.schemaregistry.metrics;
 
 import org.apache.kafka.common.MetricName;
+import org.apache.kafka.common.metrics.MeasurableStat;
 import org.apache.kafka.common.metrics.Metrics;
 import org.apache.kafka.common.metrics.Sensor;
 import org.apache.kafka.common.metrics.stats.Value;
 
 import java.util.concurrent.atomic.AtomicLong;
 
 public class SchemaRegistryMetric {
+  @Deprecated
   private final AtomicLong count = new AtomicLong();
   private final Sensor sensor;
 
+  @Deprecated
   public SchemaRegistryMetric(Metrics metrics, String sensorName, MetricName metricName) {
+    this(metrics, sensorName, metricName, new Value());
+  }
+
+  public SchemaRegistryMetric(Metrics metrics, String sensorName, MetricName metricName,
```

Comment:
Should we add some static methods for commonly used metrics like
```
public static SchemaRegistryMetric getRateMetric(Metrics metrics, String sensorName, MetricName metricName){
    sensor.add(metricName, new Rate());
}
```
If we do this then this will make adding metrics in the future much easier, since we won't have to research different types of MeasurableStat

Reply:
I got your point. I actually wanted the caller to do the research and choose the best type. You can even have compound types like `new Rate(new WindowedCount())`, e.g., [here](https://github.com/confluentinc/rest-utils/blob/460dc9f55723aeee6d67ee822d48392b7a4ab751/core/src/main/java/io/confluent/rest/metrics/MetricsResourceMethodApplicationListener.java#L194). Having the metric name and stat type on the same line makes it easy to read IMO.